source/mainnet/net/guides/create-account.rst

```diff
@@ -89,7 +89,7 @@ To learn more about identities and accounts, see :ref:`Identities <reference-id-
 
     .. Note::
 
-       It is no longer possible for users of |mw-gen1| on an Android device to create new accounts.
+       It is no longer possible for users of |mw-gen1| to create new accounts.
```

Comment:
Do we still need the full description below the note, if it is not possible to create an account anyway?

Reply:
It was copy/paste from when we did the same for Android.